tests/ui/requirements.txt

```diff
@@ -7,3 +8,9 @@ pytest-selenium==1.11.1
 pytest-variables==1.7.0
 pytest-xdist==1.20.0
 selenium==3.5.0
+hashin==0.11.2
+zope.component==4.3.0
+zope.event==4.2.0
+zope.interface==4.3.3
```

Comment:
What are they used for? Or rather, for what dependency did you add them?

Reply:
I think I was trying to pin a bunch of them to figure out some early problems.